bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Microsoft.Azure.Devices.Gateway.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
     <AssemblyName>Microsoft.Azure.Devices.Gateway</AssemblyName>
     <PackageId>Microsoft.Azure.Devices.Gateway</PackageId>
     <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
```

Comment:
Nops. I contacted the team to check what version is this, this is basically the tool that we comply, the greater the better... So, don't need to change on this case. 

---
In reply to: [112308267](https://github.com/Azure/azure-iot-gateway-sdk/pull/221#discussion_r112308267) [](ancestors = 112308267)